src/model/Surface.cpp

```diff
@@ -1027,6 +1027,8 @@ namespace model {
       double tol = 0.01;       //  1 cm tolerance
       double areaTol = 0.001;  // 10 cm2 tolerance
 
+      bool extraLogging = false;
```

Comment:
Same idea. Should be a constexpr

Reply:
constexpr done in db79456fe87054ed8957888e22f329598bdcec6f

Reply:
Thanks